test/test_lib/test_in_out/test_saver.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import lib.in_out.saver as saver
+import lib.util as util
+import lib.config as config
+import os
+import csv
+import networkx as nx
+
+current_directory = os.path.dirname(os.path.realpath(__file__))
+
+class SaverTest(unittest.TestCase):
+    
+    def setUp(self):
+        self.channel_hash = util.load_from_disk(current_directory  + '/data/channels_hash')
+        
+    def tearDown(self):
+        self.channel_hash = None
+
+    def test_check_if_dir_exists(self):
+        saver.check_if_dir_exists(current_directory + '/test/')
+        assert os.path.exists(os.path.dirname(current_directory + '/test/'))
+        os.rmdir(current_directory + '/test/')
+        
+    def test_draw_nx_graph(self):
```

Comment:
missing test here

Reply:
@prasadtalasila updated. But the test fails on travis, although it works on my local machine. It is possibly due to path misconfiguration of pygraphviz.

Reply:
@prasadtalasila fixed. Graphviz was not being installed on travis